README.md

```diff
@@ -11,12 +11,33 @@ npm i --save @gainsnetwork/trading-sdk
 ```typescript
 // import and initialize
 import { TradingSDK, SupportedChainId } from "@gainsnetwork/trading-sdk";
-const tradingSdk = new TradingSDK(SupportedChainId.ArbitrumSepolia);
+
+// To execute write transactions, it is necessary to pass the signer parameter. However, this can be omitted if you only need to generate the transaction data.
```

Comment:
maybe split Usage for

`Usage (read)` - this is current master
`Usage (build tx)` - omit signer 
`Usage (write tx)` - show example of constructing signer and sending data

what you think?